pkg/catalog/manager/traverse.go

```diff
@@ -258,10 +281,7 @@ func (m *Manager) traverseAndUpdate(helm *helmlib.Helm, commit string, cmt *Cata
 		commit = ""
 	}
 
-	v3.CatalogConditionUpgraded.True(catalog)
```

Comment:
don't move that

Reply:
In order for the templates that successfully created to not falsely show up in the logs, we have to set upgrade and discached true  even if there are errors, so it does in fact need to move